.travis.yml

```diff
@@ -4,13 +4,9 @@ sudo: false
 install:
   - pip install tox codecov
 
-env:
-  - TOX_ENV=flake8
-  - TOX_ENV=check-manifest
-  - TOX_ENV=pypy-tests
-  - TOX_ENV=py27-tests
-  - TOX_ENV=py33-tests
-  - TOX_ENV=py34-tests
+before_script:
+  - git remote set-branches --add origin master
+  - git fetch origin master
```

Comment:
Maybe `check` should do this automatically? Otherwise I'm guessing some overwhelming percentage of `check` users will need to copy-paste this exact thing into their CI scripts.